src/main/java/ui/UI.java

```diff
@@ -307,7 +307,7 @@ private void setupMainStage(Scene scene) {
         mainStage.setTitle("HubTurbo " + Utility.version(VERSION_MAJOR, VERSION_MINOR, VERSION_PATCH));
         mainStage.setScene(scene);
         mainStage.show();
-        mainStage.setOnCloseRequest(e -> quit());
+        mainStage.setOnCloseRequest(e -> quit(false));
```

Comment:
I think having a constant `USER_LOGOUT` which value is `true` will help readability as to why we need to pass a parameter to `quit`. I am not sure where can I put such constant